features/0453-issue-credential-v2/README.md

```diff
@@ -303,19 +307,34 @@ Message Format:
     "more_available": "<count>",
     "formats" : [
         {
-            "attach_id" : "<attach@id value>",
+            "attach_id" : "<attachment identifier>",
             "format" : "<format-and-version>",
         }
     ],
     "credentials~attach": [
         {
-            "@id": "<attachment-id>",
+            "@id": "<attachment identifier>",
```

Comment:
Are you taking into account the 3 types of attachments? For example, if the attachment type is "link", the inclusion of a hash is automatic.  I'm wondering if we should minor-version update the attachment RFC to include an optional hash when base64url encoding is used.

I would think that we might also want to talk about the option of signing the attachment, as is possible with the attachment RFC.  That would ensure that the supplement is signed in a way that both the holder and subsequent verifiers can verify.